python/tvm/relay/op/strategy/mali.py

```diff
@@ -105,6 +146,16 @@ def conv2d_winograd_without_weight_transfrom_strategy_mali(attrs, inputs, out_ty
             wrap_topi_schedule(topi.mali.schedule_conv2d_nchw_winograd),
             name="conv2d_nchw_winograd.mali",
         )
+    elif layout == "NHWC":
+        if is_auto_scheduler_enabled():
+            strategy.add_implementation(
+                wrap_compute_conv2d(topi.nn.conv2d_winograd_nhwc_without_weight_transform),
+                naive_schedule,  # this implementation should never be picked by autotvm
+                name="conv2d_nhwc_winograd_without_weight_transform",
+                plevel=15,
+            )
+        else:
+            logger.error("AutoTVM doesn't support NHWC winograd on Mali currently")
```

Comment:
Make it consistent as other cases.
```suggestion
        if not is_auto_scheduler_enabled():
            logger.error("Winograd conv2d NHWC is not enabled for mali without auto_scheduler")
        strategy.add_implementation(
            wrap_compute_conv2d(topi.nn.conv2d_winograd_nhwc_without_weight_transform),
            naive_schedule,  # this implementation should never be picked by autotvm
            name="conv2d_nhwc_winograd_without_weight_transform",
            plevel=15,
        )
```